package.json

```diff
@@ -34,8 +34,8 @@
     "lint:js": "eslint --quiet --ext .js --ext .jsx .",
     "lint:circle": "eslint --ext .js --ext .jsx -c circle.eslint.json .",
     "lint:js:fix": "eslint --fix --quiet --ext .js --ext .jsx .",
-    "lint:js:changed": "LIST=`git diff-index --diff-filter=d --name-only --cached HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --quiet $LIST; fi",
-    "lint:js:changed:fix": "LIST=`git diff-index --diff-filter=d --name-only HEAD | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --fix --quiet $LIST; fi",
+    "lint:js:changed": "LIST=`git status --porcelain | awk 'match($1, \"[AM]\"){print $2}' | grep \"\\.js[x]\\{0,1\\}$\"`; if [ \"$LIST\" ]; then eslint --quiet $LIST; fi",
```

Comment:
Previously, this command lints all files that are staged. (The name is misleading, granted.)

This change will trigger linting on all files that have been changed, not just on those which are staged.

I just changed a random file and tested the two commands and you can see the difference. (Note: I have an untracked `.dependency-cruiser.js`.)
![image](https://user-images.githubusercontent.com/12970166/104774129-f83e2f80-572a-11eb-9872-3b17fe48d8aa.png)

Reply:
Maybe we need two commands?

In my typical workflow, I don't explicitly call `git add`. That happens automatically I guess? 

I'd like to have the ability to lint files I've modified without having to explicitly call `git add`.

Reply:
For personal workflows, I recommend creating a bash / shell alias for yourself. We already have _waaaaaay_ too many NPM scripts in there. I'd rather not add yet another unless it's going to be widely useful. For this _particular_ workflow, I recommend setting up your editor or IDE to display linting errors so you don't have to run a separate command. For more information on how to do that, check out ESLint's [Integrations](https://eslint.org/docs/user-guide/integrations) page.

All that said, I leave it to the fine folks on the @department-of-veterans-affairs/frontend-review-group since this is no longer my wheelhouse. :smile: